src/docs/sdk/sessions.mdx

```diff
@@ -171,10 +172,10 @@ known.
 
 ### Session Counting / init
 
-It's crucial that the intial session update sent to the system has `init` set
+It's crucial that the initial session update sent to the system has `init` set
 to `true`. This is necessary because the server currently does not
-deduplicate the total session count as an optimization.  If a session is sent
-with `init` not set once.
+deduplicate the total session count as an optimization.  If the initial
```

Comment:
```suggestion
deduplicate the total session count as an optimization. If the initial
```

Reply:
:-D you don’t like these double-spaces? Neither does my auto-formatter btw, so I just reformatted everything.